mlir/test/CMakeLists.txt

```diff
@@ -36,6 +37,7 @@ message(STATUS "MLIR_CONV_VALIDATION_WRAPPER_LIBRARY_DIR: ${MLIR_CONV_VALIDATION
 # Need to explicitly give paths to llvm tools
 set(LLVM_LIT_TOOLS_DIR ${LLVM_EXTERNAL_BIN_DIR})
 set(LLVM_EXTERNAL_LIT "${LLVM_LIT_TOOLS_DIR}/llvm-lit")
+set(LLVM_DEFAULT_EXTERNAL_LIT "${LLVM_LIT_TOOLS_DIR}/llvm-lit")
```

Comment:
I need this variable to ensure it looks for llvm-lit in the external bin directory and not build/bin.

Reply:
Do you still need this?